NGS2_WITNESS_Cycle1_bayesian_exp1.R

```diff
@@ -0,0 +1,168 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. in August 2017
+
+#Summary Function
+
+bayesian.summary <- function (model) {
+  prior.summary<-prior_summary(model)
+  print(prior.summary)
+  result.summary<-summary(model, digits =3)
+  print(result.summary)
+  plot(model, plotfun = "areas", prob = 0.9)
+  }
+
+#Hypothesis 4.1.1: Fixed network compositions reduce cooperation
+#Rand  et al. (2011) coeff = -0.186, SE = 0.036, p<.000
+
+glm.out.4.1.1.default=stan_glm(decision~round_num, 
+                               data=exp1_cooperation.fixed,
+                               family = binomial(link = "logit"), 
+                               chains = 3, iter = 3000)
+bayesian.summary(glm.out.4.1.1.default)
+#launch_shinystan(glm.out.4.1.1.default)

```

Comment:
This is a pretty heavy web-based application to explore results. I wanted to leave it there to remind ourselves it's available if a reviewer asks for more in-depth diagnostics, but want to avoid launching it unless we really want to.